framework/src/modules/http_api/defaults/config.js

```diff
@@ -201,6 +222,13 @@ const defaultConfig = {
 				whiteList: ['127.0.0.1'],
 			},
 		},
+		apm: {
+			enabled: true,
```

Comment:
Maybe it makes sense to enable that by default for devnet/alphanet only?

Reply:
I have disabled it for `Testnet` and `Mainnet` by default https://github.com/LiskHQ/lisk-core/pull/139